templates/vs/csharp/custom-copilot-travel-agent/m365agents.local.yml.tpl

```diff
@@ -0,0 +1,146 @@
+# yaml-language-server: $schema=https://aka.ms/m365-agents-toolkits/v1.11/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.11
+
+provision:
+  # Creates a Teams app
+  - uses: teamsApp/create
+    with:
+      # Teams app name
+      name: {{appName}}${{APP_NAME_SUFFIX}}
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile:
+      teamsAppId: TEAMS_APP_ID
+
+  # Create AAD app for OAuth connection
+  - uses: aadApp/create
+    with:
+      # The AAD app's display name
+      name: {{appName}}-AAD${{APP_NAME_SUFFIX}}
+      generateClientSecret: true
```

Comment:
If this is for single bot registration, I believe you'll need to generate service principal for it

Reply:
added